static/js/giphy.js

```diff
@@ -99,6 +108,44 @@ function update_grid_with_search_term() {
     return undefined;
 }
 
+export function hide_giphy_popover() {
```

Comment:
Can you add a comment documenting that this returns whether the popover had been open?

Reply:
Done.